commands/setup-dorothy

```diff
@@ -353,6 +353,12 @@ setup_shell() {
 	check "$HOME/.bashrc"
 	check "$HOME/.zshrc"
 	check "$HOME/.zprofile"
+    if ! test -z "${ZDOTDIR}"; then
+	    check "$ZDOTDIR/.zshrc"
+	    check "$ZDOTDIR/zshrc"
+	    check "$ZDOTDIR/.zprofile"
+	    check "$ZDOTDIR/zprofile"
+    fi
```

Comment:
Why is there checks for with and without dot prefixes?

Reply:
Because one doesnt need to have the dot infront of the file if using zdotdir. This varies between programs and with zsh both works with zdotdir, ie with or without dot.

Reply:
But i guess then a regex on file name would be more appropriate than duplicate check

Reply:
also on a general note it seems that one can omit the period for many config files when they are located under `.config/` but I don't think this is true for all programs config files.

Reply:
https://zsh.sourceforge.io/Intro/intro_3.html seems the dot prefix is necessary, no documentation there about its omission

Reply:
Hmm interesting. I am still pretty sure it would work without the dots when put under config home.